docs/compatibility.md

```diff
@@ -504,12 +506,12 @@ Here are some examples of regular expression patterns that are not supported on
 - Empty groups: `()`
 - Regular expressions containing null characters (unless the pattern is a simple literal string)
 - Beginning-of-line and end-of-line anchors (`^` and `$`) are not supported in some contexts, such as when combined 
-- with a choice (`^|a`) or when used anywhere in `regexp_replace` patterns.
+  with a choice (`^|a`) or when used anywhere in `regexp_replace` patterns.
 
 In addition to these cases that can be detected, there is also one known issue that can cause incorrect results:
 
-- `$` does not match the end of a string if the string ends with a line-terminator 
-  ([cuDF issue #9620](https://github.com/rapidsai/cudf/issues/9620))
```

Comment:
This issue was resolved in https://github.com/NVIDIA/spark-rapids/pull/4239 but the docs did not get updated. I can raise a separate PR for this change if necessary.